### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by Rakoski**
Pls accept pull

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,76 @@
...
+
+1. **Get Pipelines**: Retrieve a list of pipelines.
+
+2. **Get Pipelines By Company**: Fetch pipelines associated with a specific company.
+
+3. **Get Random Pipelines**: Get a random set of pipelines.
+
+4. **Remove Experience on LinkedIn**: Remove existing experience entries on LinkedIn profiles.
+
+5. **Add LinkedIn Experience**: Add new experience entries to LinkedIn profiles.
```
Comment (Lewin-B) [inline README.md]: I'm not sure what this means but I would reword it so that its clear that our changes only effect our site

File: README.md [hunk 2]
```diff
@@ -1 +1,76 @@
...
+
+6. **Get Profiles**: Retrieve a list of LinkedIn profiles.
```
Comment (Lewin-B) [inline README.md]: Same as my first comment

File: README.md [hunk 3]
```diff
@@ -1 +1,76 @@
...
+
+7. **Delete Profiles**: Remove LinkedIn profiles from the system.
```
Comment (Lewin-B) [inline README.md]: Same as my first comment

File: README.md [hunk 4]
```diff
@@ -1 +1,76 @@
...
+
+8. **Update Profiles**: Modify existing LinkedIn profiles.
```
Comment (Lewin-B) [inline README.md]: Same as my first comment

File: README.md [hunk 5]
```diff
@@ -1 +1,76 @@
...
+
+## Technologies Used
```
Comment (Lewin-B) [inline README.md:25]: I would add Node.js to technologies used as well

**Review by Lewin-B — CHANGES_REQUESTED** [review]
I love it and thank you so much, Just left a few comments on some things but again thank you

File: README.md [hunk 6]
```diff
@@ -8,19 +8,19 @@ This project is a web application that provides functionality to manage pipeline
...
+4. **Remove Experience on LinkedIn**: Remove existing experience entries on pipelines' LinkedIn profiles.
 
-5. **Add LinkedIn Experience**: Add new experience entries to LinkedIn profiles.
+5. **Add LinkedIn Experience**: Add new experience entries to pipelines' Linkedin profiles.
 
-6. **Get Profiles**: Retrieve a list of LinkedIn profiles.
+6. **Get Profiles**: Retrieve a list of user profiles and their LinkedIn companies & potential experience from pipelines.lol.
 
-7. **Delete Profiles**: Remove LinkedIn profiles from the system.
+7. **Delete Profiles**: Remove user profiles and their LinkedIn profiles from the pipeilnes.lol.
```
Comment (Lewin-B) [inline README.md]: I would get rid of the "The" so it either says "from pipelines.lol" or "from the pipelines.lol webpage" also pipelines is mispelled

**Review by Lewin-B — CHANGES_REQUESTED** [review]
One last thing

**Review by Lewin-B — APPROVED** [review]
Looks Good to me!

**Comment by Rakoski** [comment]
Thanks man! I still need another person to review it for it to actually enter the repo

**Comment by aidanouckama** [comment]
Hi Rako!

Unfortunately, we are in no need for a contribution for our README file. We appreciate the contribution but we will not be pulling your changes.

- Aidan
